src/components/client/common/upcomingEvents/upcomingEvents.css

```diff
@@ -0,0 +1,42 @@
+.b-full {
+  height: auto;
+}
+
+.text-color {
+  color: var(--darkBlue);
+  font-weight: 800;
+}
+
+.shadow {
+  box-shadow: 10px 10px 8px #888888;
+  border-radius: 20px;
+  font-family: 'Roboto';
+    font-style: Bold;
+}
+
+.image{
+    height: 400px;
+    background-image:url("src/assets/images/samplePost1.jpeg");
```

Comment:
Use webp instead of jpeg because WebP images can help you improve your website's performance, user experience, and search engine visibility.